### PR TITLE
update spider-gazelle version and replaced orm with pg-orm

### PR DESCRIPTION
**Pull request by naqvis**
This PR contains the following updates to `spider-gazelle`:

* Update dependencies to latest version
* Removed dependency on `granite` ORM
* Added `pg-orm` as a replacement ORM

### CODE REVIEW

File: frameworks/Crystal/spider-gazelle/spider-gazelle.dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM crystallang/crystal:0.27.0
+FROM placeos/crystal
```
Comment (nbrady-techempower) [inline frameworks/Crystal/spider-gazelle/spider-gazelle.dockerfile:1]: Can you tie this down to a specific version tag like 1.6.1

Comment (stakach) [inline frameworks/Crystal/spider-gazelle/spider-gazelle.dockerfile:1]:
```suggestion
FROM 84codes/crystal:1.7.2-alpine
```

Comment (stakach) [reply frameworks/Crystal/spider-gazelle/spider-gazelle.dockerfile:1]: the PlaceOS image is based on the 84codes image

**Review by stakach — APPROVED** [review]
LGTM

**Pull request merged**